bin/run-command-on-git-revisions

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
I don't feel strongly about this, but do we really need bash or would any POSIX-compliant shell work here?